client/my-sites/plans-features-main/media-queries.tsx

```diff
@@ -0,0 +1,31 @@
+import { css } from '@emotion/react';
+import type { SerializedStyles } from '@emotion/react';
+
+const sidebarWidth = 272; //in px
```

Comment:
Good point. You'd think something like this would have been defined centrally somewhere already. I'm not sure if `client/layout/sidebar/index.jsx` is the right place. This merits some investigation, so I'll drop it into a follow-up issue.

Reply:
I will follow up to create an issue here once we wrap up a few other related refactors